src/api/helpers/create-full-timestamp-from-day-month.test.ts

```diff
@@ -0,0 +1,34 @@
+import createTimestampFromNumbers from './create-full-timestamp-from-day-month';
+
+describe('api/helpers/create-date-from-numbers', () => {
+  const day = '1';
+  const month = '12';
+
+  describe('when day is not provided', () => {
+    it('should return null', () => {
+      // @ts-ignore
+      const result = createTimestampFromNumbers(undefined, month);
+
+      expect(result).toEqual(null);
+    });
+  });
+
+  describe('when month is not provided', () => {
+    it('should return null', () => {
+      // @ts-ignore
```

Comment:
We should make the day and month params option fields (`day?: string, month?: string`), then we don't need to add TS ignores